### PR TITLE
added capitalise in cpp

### PR DESCRIPTION
**Pull request by Bhanu0202**
Please make sure you have done the following:

- [x] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
- [x] Added your Codes
- [ ] Tested your codes
- [x] Added credits (author and date) at the top of the code
- [x] Updated README.md
- [x] Added yourself as a contributor using `all-conributors-cli`

### CODE REVIEW

**Comment by Bhanu0202** [comment]
@MadhavBahlMD The code does capitalise first letter of each word. Please check it again. Also, let me know if I am still wrong.Thanks :)

File: day6/cpp/anagrams.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	cin >> sa >>sb;
+	int la = sa.length();
+	int lb = sb.length();
+	int count[26] = {0};
+	if(la != lb){
+	    cout << "Invalid Input";
+	}
+	else{
+	    int flag = 0, i;
+	    transform(sa.begin(), sa.end(), sa.begin(), ::tolower);
```
Comment (ashwek) [inline day6/cpp/anagrams.cpp]: Dont convert strings to lower case.
README file says **Do not ignore the letter case**, so _**A**bc_ and _**a**bc_ are not anagrams.

File: day6/cpp/Capitalise.cpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+int main() {
+	// your code goes here
+	string sa;
+	getline(cin, sa);
+	int l = sa.length();
+	sa[0] -= 32;
+	for(int i = 0; i < l; i++){
+	    if(sa[i] == 32 && sa[i+1] != 32){
+	        sa[i + 1] -= 32;
```
Comment (ashwek) [inline day6/cpp/Capitalise.cpp]: before subtracting 32 please check if `sa[i + 1]` is a lower case alphabet or not. If it is an upper case letter, subtracting 32 will give incorrect answer.

`Input :`**Abc**
`Output :`**!bc**

File: day6/cpp/anagrams.cpp (hunk 1 above)
Comment (Bhanu0202) [reply day6/cpp/anagrams.cpp]: Thanks for the correction. I will update it soon.

File: day6/cpp/Capitalise.cpp (hunk 2 above)
Comment (Bhanu0202) [reply day6/cpp/Capitalise.cpp]: Thanks for the correction. I will update it soon.

**Pull request merged**